### ISSUE TITLE
Can't get OutputFileName from EnvDTE

### ISSUE THREAD

**Issue by bricelam** [dotnet/project-system#667]
Similar to #665, I used to be able to do the following.

``` C#
var outputFileName = project.Properties.Item("OutputFileName")?.Value as string;
```

However, the `OutputFileName` property doesn't exist on .NET Core projects.

**Comment by bricelam** [comment]
### Workaround
```C#
var context = (IVsBrowseObjectContext)project;
var unconfiguredProject = context.UnconfiguredProject;
var configuredProject = await unconfiguredProject.GetSuggestedConfiguredProjectAsync();
var properties = configuredProject.Services.ProjectPropertiesProvider.GetCommonProperties();
var outputFileName = await properties.GetEvaluatedPropertyValueAsync("TargetFileName");
```

**Comment by davkean** [comment]
This has been fixed in 15.8 Preview 3.
